configs/buildbot-master.cloud-init

```diff
@@ -10,4 +10,7 @@ ssh_pwauth: true
 moz_instance_type: {moz_instance_type}
 # buildbot masters use their first ephemeral disk for swap
 mounts:
- - [ ephemeral0, /mnt, auto, "defaults,noexec" ]
+ - [ ephemeral0, none, swap, sw, 0, 0 ]
+bootcmd:
+ - mkswap /dev/xvdb
```

Comment:
Well, that doesn't work.  If cloud-init did any logging, I might know why, but it doesn't.